docs/release-notes/v1.0.rst

```diff
@@ -1,6 +1,152 @@
+-----------------------
 
-- No changes yet
+Summary
+~~~~~~~
+
+- Released Oct 7th, 2022
+- 10 contributors
+- 78 pull requests since v0.20.1 (not all listed below)
+
+New features
+~~~~~~~~~~~~
+
+This new release is the Long Term Stable (LTS) version 1.0. Most of the changes are in the package
+infrastructure. There are no API-breaking changes introduced since 0.20.1. A number of improvements
+and bug correction have been implemented.
+A new version of g.a.d.f. has been released (v0.3). It is expected to be compatible with this
+Gammapy version.
+
+*documentation*
+
+- The tutorial gallery now relies on Sphinx gallery. The tutorials are python scripts with specific
+syntax for the description cells. This simplifies the documentation build. This will make the history
+cleaner and tutorial code reviews easier. The download button is now moved at the end of the tutorial.
+Binder is now working again for tutorials.
+
+*gammapy.data and gammapy.irf*
+
+- Support for HAWC data has been improved. In particular, a `RecoPSFMap` has been added in IRF
+to support PSF in reco energy.
+
+*gammapy.maps*
+
+- A reprojection method has been implemented on `Map` to allow for reprojection on a new `MapGeom`
+object. It supports different spatial geometries but requires identical non-spatial axes. See
+`Map.reproject_to_geom()`.
+
+*infrastructure*
+
+- The compliance with the astropy affiliated packages has been improved. In particular, tox is now
+used for testing and CI.
+
+API changes
+~~~~~~~~~~~
+
+*gammapy.modeling*
+
+- `LightCurveTemplateTemporalModel` internally relies on a `RegionMap` and does not take a table on
+`__init__` anymore. To create one from a `Table` object, users have to go through `LightCurveTemplateTemporalModel.read()`.
+
+*gammapy.stats*
+
+- For consistency with the definition of `errn` and `errp`, whic are always positve quantities,
+the sign of the value returned by `CountsStatistic.compute_errn` has been changed.
+
+Bug fixes and improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*gammapy.astro*
+
+- The DM annihilation spectral model `DarkMatterAnnihilationSpectralModel` can now be serialized
+to yaml.
+
+*gammapy.data*
+
+- An `Observation.copy()` method has been implemented to allow modification of existing `Observation`
+objects.
+
+*gammapy.datasets*
+
+- The dataset name is now serialized with the Dataset as the `NAME` keyword in the primary HDU.
+- A `peek` method is now available for the `MapEvaluator` to help debugging issues
+with model evaluation on a `Dataset`.
+
+*gammapy.makers*
+
+- Add support for drifting instrument on the `MapDatasetMaker`.
+
+*gammapy.maps*
+
+- `RegionNDMap.sample_coord()` has been implemented to generate events from a region map.
+
+*gammapy.modeling*
+
+- A bug has been corrected in `TemporalModel.sample_time()`.
+- `TemporalModel.integrate` now provides a generic integration method.
+- A new `TemplatePhaseCurveTemporalModel` has been added to support pulsar-like lightcurves.
+
```

Comment:
I suppose it will be good to mention that the serialised covariance matrix does not have the parameters as column names any more. This does not affect our internal mechanism, but in case someone was using it to know the parameters list..